packages/create-modular-react-app/src/__tests__/repo-integrity.test.ts

```diff
@@ -1,15 +1,7 @@
 import createModularApp from '..';
-import execa from 'execa';
 import path from 'path';
 import tmp from 'tmp';
-
-function modular(str: string, cwd: string, opts: Record<string, unknown> = {}) {
-  return execa('yarnpkg', ['modular', ...str.split(' ')], {
-    cwd,
-    cleanup: true,
-    ...opts,
-  });
-}
+import { runModular } from 'modular-scripts/src/test/utils';
```

Comment:
the util file containing the function is in modular-scripts, while it's being called in create-modular-react-app

Reply:
We don't have that dependency in `package.json` though (and we probably don't want to have it). We will probably factor out common utils when we do modularised Modular - since runModular is three lines anyway, can we just duplicate it here?